subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -24,6 +24,11 @@
 
 #include "ll_test.h"
 
+#define LOG_MODULE_NAME bt_ctlr_llsw_nordic_lll_adv
```

Comment:
Is this and the 2 includes below part of this patch? Not sure those are relevant here

Reply:
I will clear it and squash commits.